mifosng-provider/src/main/java/org/mifosplatform/infrastructure/configuration/domain/GlobalConfigurationProperty.java

```diff
@@ -72,12 +73,24 @@ public boolean updateTo(final boolean value) {
         }
 
         final String valueParamName = "value";
+        final Long previousValue = this.value;
         if (command.isChangeInLongParameterNamed(valueParamName, this.value)) {
             final Long newValue = command.longValueOfParameterNamed(valueParamName);
             actualChanges.put(valueParamName, newValue);
             this.value = newValue;
         }
-
+        
+        Boolean exception = false;
+        if(this.enabled == true && command.hasParameter(valueParamName) && this.value == 0 ){
+        	exception = true;
+        }else if(this.enabled == true && !command.hasParameter(valueParamName) && previousValue == 0){
+        	exception = true;
+        }
+        
+        if(exception){
+        	final String errorMsg = "Force Password Reset days value must be greater than 0.";
+            throw new ForcePasswordResetException(errorMsg, "password.reset.days.value.must.be.greater.than.zero");
```

Comment:
Sine you have a new Exception, encapsulate error message and code within the exception class.

So here you would simply throw new ForcePasswordResetException();